src/components/FarmLPCard/FarmLPCardDetails.tsx

```diff
@@ -296,164 +324,202 @@ const FarmLPCardDetails: React.FC<{
       ? '< $0.001'
       : '$' + earnedUSD.toLocaleString();
 
+  const valueOfTotalStakedAmountInUSDC =
+    valueOfTotalStakedAmountInBaseToken &&
+    USDPrice?.quote(valueOfTotalStakedAmountInBaseToken);
+
+  const tvl = valueOfTotalStakedAmountInUSDC
```

Comment:
Why is this logic duplicated with the dual farm card, we really need to merge the common logic between the two cards in the future